### ISSUE TITLE
Making it work

### ISSUE THREAD

**Issue by tiny102** [tripflex/MOD-t#10]
I have been trying to get this to work with MOD-T but it seems not to recognize it as it is plugged in via cable. I have even tried the one with WIFI disabled but that still has not worked. Is there something that I'm missing when installing the application or should I not even try to get it to work again

**Comment by GeneTognetti** [comment]
Not sure what to suggest. Did you try the How to print without New Matter section? Good luck!

**Comment by tiny102** [comment]
No, I have not I look into that then. If it does not work then should I not even worry about trying to get it to work because you cannot get it to read any of the files.

**Comment by GeneTognetti** [comment]
I was able to get mine to work following the instructions on this page -
but that was several years ago. I suggest you really look through what is
in the MOD-t group. That's what helped me. I don't know what else to
suggest. Good luck!

**Comment by tiny102** [comment]
OK thank you so much for that info
